request.go

```diff
@@ -747,7 +758,7 @@ func (r *Request) Execute(method, url string) (*Response, error) {
 		Retries(r.client.RetryCount),
 		WaitTime(r.client.RetryWaitTime),
 		MaxWaitTime(r.client.RetryMaxWaitTime),
-		RetryConditions(r.client.RetryConditions),
+		RetryConditions(append(r.client.RetryConditions, r.retryConditions...)),
```

Comment:
@rohitkg98 If we are adding a Retry condition feature at the request level; is it a request retry condition that should take place first instead of the client?

@moorereason your thoughts!

Reply:
@rohitkg98 Basically, Request retry conditions have to execute first and then client instance level. So we need to flip the append arguments.

Reply:
I am looking for this feature as well

Reply:
changed to check request specific conditions before client ones